src/targets/gpu/kernels/include/migraphx/kernels/pooling.hpp

```diff
@@ -133,9 +133,8 @@ struct window
         auto win_start = generate_array<diff_int>(rank{}, [&](auto j) {
             diff_int w   = win[j];
             diff_int dim = i[j];
+            (void)w;
             MIGRAPHX_ASSERT(w >= 1);
-            if(w == 1)
-                return dim;
```

Comment:
Wouldn't it be better to change this to `return dim * s` instead of removing the condition?

Reply:
If the rest of the following lines do the same thing.. why add an exception?